### DISCUSSION TITLE
Mobile support

### DISCUSSION THREAD

**Discussion by hbasaraner** [nick-keller/react-datasheet-grid#261] [Feature requests]
Hello all. Is there any development for mobile support of this package? I created an Excel file importer app with this package and it works on mobile browser but there is no interaction like selecting multiple cells and deleting rows. Is it ok for pull request?

**Comment by nick-keller** [comment]
Before jumping into code, let's discuss what would need to be done on how to support mobile!
Please provide a list of issues, and potential solutions for each (feel free to screenshot mobile apps that already do it to get a feel of what should be done)

**Comment by hbasaraner** [comment]
Here is some examples:

I can select cells and copy values between them. Also, I can delete selected rows in desktop:

https://user-images.githubusercontent.com/7539129/218959776-85230017-3e1e-45f2-906c-3cd3ab8a6747.mov


But I can't do same interactions on mobile:

https://user-images.githubusercontent.com/7539129/218959966-80fc83cf-da6a-44d3-a0fb-963348b44489.mov


## Solution ideas: 

1. For selecting cells, we can create dragging points and use Touch events like Google Sheet:

![WhatsApp Image 2023-02-15 at 10 27 54](https://user-images.githubusercontent.com/7539129/218960815-5828647b-c7cb-4ecc-8a35-8185362924f6.jpeg)


2. For right-click events, double tapping on holding touch like a second might be useful.

**Comment by matveev-ka** [comment]
On mobile, I need to click twice into the field to bring up the keyboard and get into the field. Can you add an option when user needs to click only once to start typing?


https://user-images.githubusercontent.com/100426632/227887209-707c8b1a-eb66-4cf6-9e14-09d6159bdb21.mp4

**Comment by matveev-ka** [comment]
On mobile unable to paste data from table formats (on video from excel). Can you fix this?


https://user-images.githubusercontent.com/100426632/227890900-46e63a31-822b-4e58-8eca-2d3a9af7fc9d.MP4

**Comment by matveev-ka** [comment]
On mobile, the dragger to pull down and copy fields doesn’t work.


https://user-images.githubusercontent.com/100426632/228166399-4abb7ab5-1157-4dfe-8e3f-2b6ca34c606e.mp4

**Comment by nick-keller** [comment]
Hello everyone,
Unfortunately I don't have the bandwidth to work on this, but I can support anyone willing to do it.
